internal/rootcoord/dml_channels.go

```diff
@@ -307,5 +322,9 @@ func (d *dmlChannels) removeChannels(names ...string) {
 }
 
 func genChannelName(prefix string, idx int64) string {
+	Params := &paramtable.Get().CommonCfg
+	if Params.PreCreatedTopicEnabled.GetAsBool() {
+		return Params.TopicNames.GetAsStrings()[idx]
```

Comment:
recommend add a new method to return pre-created channel names, "gen" means "generate", but here it doesn't generate any new name